src/rpc/blockchain.cpp

```diff
@@ -1926,6 +1932,428 @@ static UniValue savemempool(const JSONRPCRequest& request)
+    }
+    bool GetCScript(const CScriptID &hash, CScript& redeemScriptOut) const override {
+        // return a dummy TX_WITNESS_V0_KEYHASH script
+        redeemScriptOut = CScript() << OP_0 << std::vector<unsigned char>(20);
+        return true;
+    }
+};
+
+UniValue scantxoutset(const JSONRPCRequest& request)
+{
+    if (request.fHelp || request.params.size() < 1 || request.params.size() > 3)
+        throw std::runtime_error(
+            "scantxoutset <action> <scanobjects> (<options>)\n"
+            "\nScans the unspent transaction output set for possible entries that matches common scripts of given public keys.\n"
+            "\nArguments:\n"
+            "1. \"action\"                       (string, required) The action to execute\n"
+            "                                      \"start\" for starting a scan\n"
+            "                                      \"abort\" for aborting the current scan (returns true when abort was successful)\n"
+            "                                      \"status\" for progress report (in %) of the current scan\n"
+            "2. \"scanobjects\"                  (array, required) Array of scan objects (only one object type per scan object allowed)\n"
+            "      [\n"
+            "        { \"address\" : \"<address>\" },       (string, optional) Bitcoin address\n"
+            "        { \"script\"  : \"<scriptPubKey>\" },  (string, optional) HEX encoded script (scriptPubKey)\n"
+            "        { \"pubkey\"  :                      (object, optional) Public key\n"
+            "          {\n"
+            "            \"pubkey\" : \"<pubkey\">,         (string, required) HEX encoded public key\n"
+            "            \"script_types\" : [ ... ],      (array, optional) Array of script-types to derive from the pubkey (possible values: \"P2PKH\", \"P2SH-P2WPKH\", \"P2WPKH\")\n"
+            "          }\n"
+            "        },\n"
+            "        { \"xpub\"  :                        (object, optional) Use an extended public key child key range (m/0/k & m/1/k) to derive scripts from\n"
+            "          { \n"
+            "            \"xpub\" : \"<xpub\">,             (string, required) Base58check encoded extended public key (xpub)\n"
+            "            \"range\" : [ <s>, <e> ],        (array, optional) Range of keys that will be deriven from the given xpubs (default is 0 to 1000)\n"
+            "            \"script_types\" : [ ... ],      (array, optional) Array of derivation type (possible values: \"P2PKH\", \"P2SH-P2WPKH\", \"P2WPKH\")\n"
+            "          }\n"
+            "        },\n"
+            "      ]\n"
+            "3. \"options\"                               (object, optional)\n"
+            "      \"rawsweep\": {                        (object, optional) Optionally creates a raw sweep transaction\n"
+            "          \"address\": \"address\",            (string, optional) Address where the funds should be sent to\n"
```

Comment:
As @Sjors point out, this should be required if `rawsweep` is set?

Reply:
Fixed